### PR TITLE
Refine deployment script for Rinkeby test network

### PR DESCRIPTION
**Pull request by BrassLion**
This change adds the Rinkeby test network as a deployment target,
as this is where our testnet multisig resides.

### CODE REVIEW

File: deploy/deployArrowTokenHot.js [hunk 1]
```diff
@@ -1,57 +1,85 @@
...
+    deployedContract = r;
+  })
+  .then(async (r) => {
+
+    // Only verify on chains that are supported by Etherscan.
+    if(chain > 6) {
+      return
+    }
+
+    console.log("Verifying " + deployedContract.address + " implementation (" + implementation + ") on Etherscan...");
```
Comment (GoodluckH) [inline deploy/deployArrowTokenHot.js]: `implementation` here is used before initialized.

**Review by GoodluckH — CHANGES_REQUESTED** [review]
Generally LGTM. One small bug to be fixed (see my comment above). Can/should we also verify the proxy contract itself, not just the implementation contract?

File: deploy/deployArrowTokenHot.js (hunk 1 above)
Comment (BrassLion) [reply deploy/deployArrowTokenHot.js]: Good catch. Have fixed.

**Comment by BrassLion** [comment]
> Generally LGTM. One small bug to be fixed (see my comment above). Can/should we also verify the proxy contract itself, not just the implementation contract?

It seems difficult to get the proxy contract to verify via the plugin, and I don't think it's as important as the token contract.

**Review by GoodluckH — APPROVED** [review]
GTG. One quick comment: should we change the contract address to in Readme to clarify addresses for the proxy and the implementation contracts?

**Comment by BrassLion** [comment]
> GTG. One quick comment: should we change the contract address to in Readme to clarify addresses for the proxy and the implementation contracts?

Yeah will do so once I've got it nailed down.

**Pull request merged**